app/assets/stylesheets/media.scss

```diff
@@ -275,9 +275,8 @@
     flex-shrink: 0;
   }
 
-  // First selector is for embargo. Last selector is other auth restrictions
-  .authLinkWrapper + .media-component-body, [data-controller="auth-restriction"]:has(.authLinkWrapper:not([hidden])) + .media-component-body {
-    max-height: calc(100vh - 88px); // Scroll height for metadata when the auth wrapper is displaying
+  [data-controller="auth-restriction"]:not([hidden]) + .media-component-body {
```

Comment:
without the selector simplification here and the associated HTML and JS updates (motivation described in more detail the last commit message), i found that the video player wouldn't expand to fill the space left by the dismissed banner.  also, the video player would be a bit jumpy on page load, and would be shrunk as if the banner was already displayed, even if the banner had yet to be shown because a response from stacks had yet to be received.

the fix associated with this CSS tweak seems to smooth things out so that the video player consistently shrinks when auth message is displayed, and expands when it's not.

example of the incorrectly unexpanded player before the fix in the last commit:
<img width="1309" alt="Screen Shot 2023-11-29 at 10 11 23 PM" src="https://github.com/sul-dlss/sul-embed/assets/7741604/72e91cb7-ed1c-413e-a65f-5d8fab212520">

Reply:
ok, after a bunch of uncertainty, i did what i thought was the safest thing to move this PR along without introducing a regression: i put the old selector back.  but i made two small tweaks: i flipped the order, because that felt more natural given the order of the DOM elements in `media_with_companion_windows_component.erb`, and i updated the comment because i'm pretty sure the selector i added back is for citation-only.

per stereotypically wordy slack reply, there's a corner case with two banners that neither the old CSS nor the slightly re-jiggered CSS seem to account for.  but i don't think this PR makes that worse, so maybe we can circle back on that in a follow up if need be.